web/README.md

```diff
@@ -0,0 +1,499 @@
+# HarbourBridge: Web APIs (Experimental)
```

Comment:
I and Nevin discussed this offline, and decided to keep this file in this PR itself since this doesn't change the main ReadMe.md that's visible to users by default.